mod/wikirate/lib/javascript/script_collapse.js.coffee

```diff
@@ -88,16 +88,16 @@ findCollapseTarget = (toggle) ->
     else
       $toggle.parent()
 
-  target = $toggle.data('target') || '.collapse'
+  if $toggle.attr("href") != null
+    return $($toggle.attr("href"))
+  target = $toggle.attr("href") || $toggle.data('target') || '.collapse'
   if $toggle.data("collapse")?
     target += $toggle.data("collapse")
-
-  if $toggle.find(target).length
+  else if $toggle.find(target).length
```

Comment:
I checked the bootstrap docs. They use `href` for links and `data-target` for buttons so we should support both.